pkg/kube/inject/app_probe.go

```diff
@@ -80,18 +80,53 @@ func convertAppProber(probe *corev1.Probe, newURL string, statusPort int) *corev
 	return nil
 }
 
+// patchHTTPGetAction patches a HTTPGet action with given URL and port.
+// Also patch the scheme to HTTP if the scheme is HTTPS
+// as pilot agent uses https to request application endpoint.
+func patchHTTPGetAction(action *corev1.HTTPGetAction, url string, port int) {
```

Comment:
maybe call rewriteHTTPGetAction.

And can you reuse this in convertAppProberHTTPGet.

Reply:
@hzxuzhonghu will rename it, it's used in rest of the places places including `convertAppProberHTTPGet` already though:
https://github.com/istio/istio/blob/69181feb10dc3a29954944f7b407fa4f4e901103/pkg/kube/inject/app_probe.go#L112